web_widget_x2many_2d_matrix/README.rst

```diff
@@ -107,7 +114,14 @@ the field in the default function::
 
 Now in our wizard, we can use::
 
-<field name="task_ids" widget="x2many_2d_matrix" field_x_axis="project_id" field_y_axis="user_id" field_value="planned_hours" />
+ <field name="task_ids" widget="x2many_2d_matrix" field_x_axis="project_id" field_y_axis="user_id" field_value="planned_hours">
+     <tree>
+         <field name="task_ids"/>
```

Comment:
At least I am getting this error when I tried to use this approach in https://github.com/OCA/stock-logistics-warehouse/pull/303